templates/docs/examples/patterns/equal-height-row/4-items-per-column.html

```diff
@@ -7,94 +7,92 @@
 
 <div class="p-equal-height-row has-divider-3">
   <div class="p-equal-height-row__col">
-    <div class="p-equal-height-row__item">
-      <div class="p-media-container">
-        <img width="320"
-          src="https://assets.ubuntu.com/v1/c4176cd9-Kernelt%20industries-80%201.jpg"
-          alt="" />
-      </div>
+    <div class="p-media-container">
+      <img width="320"
+        src="https://assets.ubuntu.com/v1/c4176cd9-Kernelt%20industries-80%201.jpg"
+        alt="">
     </div>
-    <div class="p-equal-height-row__item">
-      <hr class='p-rule--highlight u-hide--medium u-hide--small'/>
+
+    <div>
+      <hr class="p-rule--highlight u-hide--medium u-hide--small">
       <h5 class="p-heading--5">Continuous security</h5>
     </div>
-    <p class="p-equal-height-row__item">
-      Imagine a world where every device is trustworthy. We designed every aspect of Ubuntu
-      Core to create the most secure embedded Linux ever, with a 10 year LTS commitment.
+
+    <p>
+      Imagine a world where every device is trustworthy. We designed-
     </p>
-    <div class="p-equal-height-row__item">
-      <a href="#">
-        <p>Learn more about Ubuntu security&nbsp;›</p>
-      </a>
-    </div>
+
+    <a href="#">
+      <p>Learn more about Ubuntu security&nbsp;›</p>
+    </a>
   </div>
+
   <div class="p-equal-height-row__col">
-    <div class="p-equal-height-row__item">
-      <div class="p-media-container">
-        <img width="320"
-          src="https://assets.ubuntu.com/v1/1624c652-Kernelt%20industries-80%201-1.jpg"
-          alt="" />
-      </div>
+    <div class="p-media-container">
+      <img width="320"
+        src="https://assets.ubuntu.com/v1/1624c652-Kernelt%20industries-80%201-1.jpg"
+        alt="">
     </div>
-    <div class="p-equal-height-row__item">
-      <hr class='p-rule--highlight u-hide--medium u-hide--small'/>
+
+    <div>
```

Comment:
That's the reason why we have the `p-equal-height-row__item` class name. If an element is required for component to work, it should have a class name, so it can be clearly identified as part of the component and comply with BEM-like naming.

Otherwise it's just too easy to just consider it redundant wrapper and remove from HTML without understanding the consequences.